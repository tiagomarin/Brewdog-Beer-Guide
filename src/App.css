@font-face {
  font-family: Veneer3;
  src: url('./assets/VeneerThree.ttf') format('truetype');
}

@font-face {
  font-family: Veneer;
  src: url('./assets/Veneer.ttf') format('truetype');
}

@font-face {
  font-family: Claredon;
  src: url('./assets/Clarendon-Regular.ttf') format('truetype');
}

*,
*::before,
*::after {
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
}

* {
  padding: 0;
  margin: 0;
}

html {
  font-size: 62.5%;
  scroll-behavior: smooth;
}

body {
  font-family: Veneer3, sans-serif, Clarendon;
  letter-spacing: 0.5vmin;
}

main {
  width: 100%;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  background-color: #44a3da;
}

h2,
h3 {
  color: #043552;
}
