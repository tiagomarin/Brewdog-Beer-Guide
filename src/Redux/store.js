import { configureStore } from '@reduxjs/toolkit';
import logger from 'redux-logger';
import beersSlice from './beersSlice';

const store = configureStore({
  reducer: {
    beers: beersSlice,
  },
  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});

export default store;
