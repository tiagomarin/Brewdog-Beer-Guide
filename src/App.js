import React from 'react';
import { Route, Routes } from 'react-router-dom';
import './App.css';
import BeerList from './Components/HomePage/BeerList';
import BeerDetails from './Components/BeerDetailsPage/BeerDetails';

function App() {
  return (
    <main>
      <Routes>
        <Route path="/" element={<BeerList />} />
        <Route path="BeerDetails" element={<BeerDetails />} />
      </Routes>
    </main>
  );
}

export default App;
