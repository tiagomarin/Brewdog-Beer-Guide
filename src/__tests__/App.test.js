import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import App from '../App';
import store from '../Redux/store';

describe('App component', () => {
  it('rendered App component', () => {
    const tree = render(
      <React.StrictMode>
        <Provider store={store}>
          <Router>
            <App />
          </Router>
        </Provider>
        ,
      </React.StrictMode>,
    );
    expect(tree).toMatchSnapshot();
  });
});
