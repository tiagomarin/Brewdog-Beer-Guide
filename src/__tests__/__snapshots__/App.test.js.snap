// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component rendered App component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main>
        <div
          class="home-page"
        >
          <header>
            <h2>
              All Beers
            </h2>
            <div
              class="filter-wrapper "
            >
              <select
                class="filter-menu"
                name="filter"
              >
                <option
                  value=""
                >
                  All Beers
                </option>
                <option
                  value="All Beers"
                >
                  All Beers
                </option>
                <option
                  value="Very low"
                >
                  Very low
                </option>
                <option
                  value="Low"
                >
                  Low
                </option>
                <option
                  value="Medium"
                >
                  Medium
                </option>
                <option
                  value="High"
                >
                  High
                </option>
                <option
                  value="Very high"
                >
                  Very high
                </option>
                <option
                  value="Ultra high"
                >
                  Ultra high
                </option>
              </select>
            </div>
          </header>
          <ul
            class="beer-list"
          />
        </div>
      </main>
      ,
    </div>
  </body>,
  "container": <div>
    <main>
      <div
        class="home-page"
      >
        <header>
          <h2>
            All Beers
          </h2>
          <div
            class="filter-wrapper "
          >
            <select
              class="filter-menu"
              name="filter"
            >
              <option
                value=""
              >
                All Beers
              </option>
              <option
                value="All Beers"
              >
                All Beers
              </option>
              <option
                value="Very low"
              >
                Very low
              </option>
              <option
                value="Low"
              >
                Low
              </option>
              <option
                value="Medium"
              >
                Medium
              </option>
              <option
                value="High"
              >
                High
              </option>
              <option
                value="Very high"
              >
                Very high
              </option>
              <option
                value="Ultra high"
              >
                Ultra high
              </option>
            </select>
          </div>
        </header>
        <ul
          class="beer-list"
        />
      </div>
    </main>
    ,
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
